base/setup/usetup/consup.c

```diff
@@ -820,4 +947,50 @@ CONSOLE_ClearStyledText(IN SHORT x,
     }
 }
 
+/**
+ * @CONSOLE_ClearStyledTextByID
+ *
+ * Removes a text string from a portion of area of the screen.
+ *
+ * @param[in]   page
+ *     The MUI (Multilingual User Interface) entry page number, as a unsigned long integer.
+ *
+ * @param[in]   TextID
+ *      The text identification number (ID), as a unsigned integer.
+ *
+ * @param[in]   Flags
+ *      The text style flag, as a unsigned integer.
+ *
+ * @return
+ *     Nothing.
+ *
+ * @remarks
+ *      Unlike CONSOLE_ClearStyledText, CONSOLE_ClearStyledTextByID clears a portion of text given its text ID,
+ *      entry page interface and the specific style flag of the text. The routine calls MUIGetEntryData() to automatically
```

Comment:
This second sentence doesn't seem necessary for the usage of this function.
If somebody is interested this can be observed by just reading the source of the function.